web/src/components/__tests__/Failure.spec.tsx

```diff
@@ -7,7 +8,7 @@ jest.mock('react-i18next')
 jest.mock('react-inlinesvg')
 
 describe('Failure', () => {
-  const mockTranslate = (key: string) => key
+  const mockTranslate = ((key: string) => key) as TFunction
```

Comment:
for me it works without casting

Reply:
I needed it in `yarn ts:check`, did you run that one?